tf_agents/trajectories/time_step_test.py

```diff
@@ -36,6 +36,26 @@ def testRestart(self):
     self.assertEqual(-1, time_step.observation)
     self.assertEqual(0.0, time_step.reward)
     self.assertEqual(1.0, time_step.discount)
+  
```

Comment:
Remove any empty spaces.

Reply:
Sorry, I'm not sure what do you mean by this, could you kindly elaborate?